packages/mobile/src/navigator/NavigationService.ts

```diff
@@ -123,11 +127,20 @@ export async function ensurePincode(): Promise<boolean> {
     return false
   }
 
-  if (pincodeType !== PincodeType.CustomPin) {
+  if (pincodeType !== PincodeType.CustomPin && pincodeType !== PincodeType.PhoneAuth) {
     Logger.error(TAG + '@ensurePincode', `Unsupported Pincode Type ${pincodeType}`)
     return false
   }
 
+  if (pincodeType === PincodeType.PhoneAuth) {
+    try {
+      await getPincodeWithBiometrics()
+      return true
+    } catch (error) {
+      Logger.warn(`${TAG}@ensurePincode`, `Retrieve PIN by biometrics error`, error)
```

Comment:
okay, i've updated to Logger.error and we can always silence it in Sentry/update it later if it gets noisy.

the chain of calls to fetch the pin with biometrics is `getPincodeWithBiometrics` -> `retrieveStoredItem` -> `Keychain.getGenericPassword`. for the user cancelled action in case of failed biometrics auth,`Keychain.getGenericPassword` will reject with a user cancelled error and this is already logged with `Logger.error` in `retrieveStoredItem`. The new error that is thrown from `retrieveStoredItem` is a generic `ErrorMessages.KEYCHAIN_STORAGE_ERROR` so in this function above, we would be sending the generic `ErrorMessages.KEYCHAIN_STORAGE_ERROR` to Sentry on top of the actual keychain error sent by `retrieveStoredItem`.....my thinking is that most errors that come from this event will be user cancelled errors but let's see in Sentry :)